src/dictionary.ts

```diff
@@ -1476,6 +1512,12 @@ export const en_dk : Lexeme[] = [
     translations: ["opnåelig, -t, -e"],
     keywords: ["complexity theory", "formal methods", "graph"]
   },
+  {
+    word: "record",
+    type: "sb.",
+    translations: ["post,  -en, -er, -erne"],
```

Comment:
I think this *post* stems from this
![image](https://github.com/SSoelvsten/cs-dict/assets/14313096/bcf093eb-a020-491b-a5ae-dc0b3a2a5b78)

That is, it is a single entry (a record) rather than the collection of entries. Maybe [fortegnelse](https://ordnet.dk/ddo/ordbog?query=fortegnelse) is more applicable? I may also be wrong; did someone else use "post"?

Reply:
I got it from Wikipedia: https://da.wikipedia.org/wiki/Programmering

> Består af felter af forskellig datatype. Denne afspejler bl.a. en post i en database-tabel.

But that's not exactly authoritative.

Reply:
Hmm, that is more authorative than me. :+1: